schemas/semantic_router.yaml

```diff
@@ -18,6 +18,6 @@ vectorizer:
   type: hf
   model: sentence-transformers/all-mpnet-base-v2
 routing_config:
-  distance_threshold: 0.3
```

Comment:
Will routingConfig still accept a distance threshold here?

Reply:
yes but it'll be a depreciation warning so this is a good catch and should be removed